src/cr/cube/crunch_cube.py

```diff
@@ -1207,11 +1130,11 @@ def index(self, weighted=True, prune=False):
     def zscore(self, weighted=True, prune=False, hs_dims=None):
         '''Get cube zscore measurement.'''
         counts = self.as_array(weighted=weighted, prune=prune)
-        total = self.margin(weighted=weighted, prune=prune)
-        colsum = self.margin(axis=0, weighted=weighted, prune=prune)
-        rowsum = self.margin(axis=1, weighted=weighted, prune=prune)
+        total = self._margin(weighted=weighted, prune=prune)
+        colsum = self._margin(axis=0, weighted=weighted, prune=prune)
+        rowsum = self._margin(axis=1, weighted=weighted, prune=prune)
 
-        if self.has_mr:
+        if self.has_mr or 'categorical_array' in self.dim_types:
```

Comment:
can you use something like `self.has_ca` ?

Reply:
will do